drat.Rproj

```diff
@@ -1,16 +0,0 @@
-Version: 1.0
-
-RestoreWorkspace: Default
-SaveWorkspace: Default
-AlwaysSaveHistory: Default
-
-EnableCodeIndexing: Yes
-UseSpacesForTab: Yes
-NumSpacesForTab: 2
-Encoding: UTF-8
-
-RnwWeave: knitr
-LaTeX: pdfLaTeX
-
-BuildType: Package
-PackageInstallArgs: --no-multiarch --with-keep.source
```

Comment:
Why was this file deleted?

Reply:
Because I made a mistake.

Reply:
Mistakes happen. I think it was mostly auto-generated. I can also make a copy of the one I have here (pre-merge).